module/Company/src/Company/Form/EditCompany.php

```diff
@@ -0,0 +1,325 @@
+            ),
+            'options' => array(
+                'label' => $translate->translate('Phone'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'nl_logo',
+            'attributes' => array(
+                'type'  => 'file',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Logo'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'en_logo',
+            'attributes' => array(
+                'type'  => 'file',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Logo'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'en_description',
+            'type' => 'Zend\Form\Element\Textarea',
+            'options' => array(
+                'label' => $translate->translate('Description'),
+                //'required' => 'required'
+            ),
+            'attributes' => array(
+                'type' => 'textarea'  
+            ),
+        ));
+        $this->add(array(
+            'name' => 'nl_description',
+            'type' => 'Zend\Form\Element\Textarea',
+            'options' => array(
+                'label' => $translate->translate('Description'),
+                //'required' => 'required'
+            ),
+            'attributes' => array(
+                'type' => 'textarea'  
+            ),
+        ));
+        
+        $this->add(array(
+            'name' => 'submit',
+            'attributes' => array(
+                'type'  => 'submit',
+                'value' => $translate->translate('Submit changes'),
+                'id' => 'submitbutton',
+            ),
+        ));
+        
+        $this->initFilters();
+
+    }
+    
+    protected function initFilters()
+    {
+        $filter = new InputFilter();
+
+
+        $filter->add(array(
+            'name' => 'name',
+            'required' => true,
+            'validators' => array(
+                array(
+                    'name' => 'string_length',
+                    'options' => array(
+                        'min' => 2,
+                        'max' => 127
+                    )
+                )
+            ),
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
+        ));
+        
+        /*$filter->add(array(
+            'name' => 'en.website',
+            'required' => true,
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
+            'validators' => array(
+            )
+        ));
+        
+        $filter->add(array(
+            'name' => 'nl.website',
+            'required' => true,
+            'filters' => array(
+                array('name' => 'StripTags'),
+                array('name' => 'StringTrim')
+            ),
```

Comment:
Het is hier wel beter om te valideren of iets een geldige URL is in plaats van het strippen van tags